pkg/driver/mount.go

```diff
@@ -272,6 +259,25 @@ func (m *S3Mounter) Mount(bucketName string, target string,
 	return nil
 }
 
+func moveOptionToEnvironmentVariables(optionName string, envName string, options []string, env []string) ([]string, []string) {
+	// Moves a parameter optionName from the options list to MP's environment variable list. We need this as options are
```

Comment:
Looks good. But shouldn't it be moved above `func`? Or am I showing my poor Go knowledge?

Reply:
No, I'm just too used to languages where you put the comment after the definition 😅